packages/manager/modules/enterprise-cloud-database/src/service/connection-details/flags/flags.constants.js

```diff
@@ -0,0 +1,30 @@
+export const COMMANDS = {
+  postgresql: {
+    connect: 'psql',
+    restore: 'pg_restore',
+  },
+};
+
+export const COMMANDS_LIST = {
+  CONNECT: 'connect',
+  RESTORE: 'restore',
+};
+
+export const ENDPOINT_TYPES = {
+  READ_ONLY: 'read-only',
+  READ_WRITE: 'read-write',
+};
+
+export const PARAMETERS = {
+  DATABASE: 'defaultdb',
+  MASKED_PASSWORD: '********',
```

Comment:
This looks like a duplicate https://github.com/ovh-ux/manager/pull/1242/files#diff-ac64ea46278fe4b1bfadf5094034a177R6

Reply:
Duplicate removed. Thanks!